index.html

```diff
@@ -0,0 +1,141 @@
+        </div>
+        </div>
+          <button type="button" class="show-works-2">
+          See this work here &#x2192
+          </button>
+        </div>
+      </div>
+
+    <div class="project-3">
+      <img src="images/project 3.png" alt="" class="project3">
+      <div class="work-3">
+ <h2 class="third-project">Project name goes here</h2>
+<div class="languages-3">
+          <button type="button" class="mark-up-3">HTML/CSS</button>
+          <button type="button" class="react-3">Ruby on Rails</button>
+          <button type="button" class="responsive-3">JavaScript</button>
+        </div>
+      </div>
+      <button type="button" class="show-works-3">
+          See this work here &#x2192
+          </button>
+         </div>
+
+          <div class="project-4">
+      <img src="images/Project 4.png" alt="" class="project4">
+      <div class="work-4">
+ <h2 class="forth-project">Project name goes here</h2>
+<div class="languages-4">
+          <button type="button" class="mark-up-4">HTML/CSS</button>
+          <button type="button" class="react-4">Ruby on Rails</button>
+          <button type="button" class="responsive-4">JavaScript</button>
+        </div>
+        <button type="button" class="show-works-4">
+          See this work here &#x2192
+          </button>
+      </div>
+         </div>
+
+<div class="project-5">
+      <img src="images/Project 5.png" alt="" class="project5">
+      <div class="work-5">
+ <h2 class="fifth-project">Project name goes here</h2>
+<div class="languages-5">
+          <button type="button" class="mark-up-5">HTML/CSS</button>
+          <button type="button" class="react-5">Ruby on Rails</button>
+          <button type="button" class="responsive-5">JavaScript</button>
+        </div>
+        <button type="button" class="show-works-5">
+          See this work here &#x2192
+          </button>
+      </div>
+         </div>
+
+    <div class="project-6">
+      <img src="images/Project 6.png" alt="" class="project6">
+      <div class="work-6">
+ <h2 class="sixth-project">Project name goes here</h2>
+<div class="languages-6">
+          <button type="button" class="mark-up-6">HTML/CSS</button>
+          <button type="button" class="react-6">Ruby on Rails</button>
+          <button type="button" class="responsive-6">JavaScript</button>
+        </div>
+        <button type="button" class="show-works-6">
+          See this work here &#x2192
+          </button>
+      </div>
+         </div>     
+    </section>
+
+    <section class="about-me">
+      <h1 class="me">About me</h1>
+      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+        Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
+    <button type="button" class="cv"><h6 class="reference">Get my CV</h6><img src="images/ic_download.png" alt=""></button>
+    <div class="lang">
```

Comment:
Kindly remove the borders of your button, to look like the one in Figma.
| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/198789141-c7373cb4-de79-4e27-b429-fabb7392e8d8.png)  | ![image](https://user-images.githubusercontent.com/42799579/198789050-ff125d94-4b10-4b53-a368-b0d321a21939.png)  |

You can update this simply by:

```css
element {
  border: 0;
}
```

Reply:
Hi @Gopxfs , I have made the changes you recommended.